docs/source/quicktour.mdx

```diff
@@ -213,114 +215,68 @@ and get tensors back. You can specify all of that to the tokenizer:
 ... )
 ```
 
-The padding is automatically applied on the side expected by the model (in this case, on the right), with the padding
-token the model was pretrained with. The attention mask is also adapted to take the padding into account:
+Read the [preprocessing](./preprocessing) tutorial for more details about tokenization.
+
+### Using the AutoModel
+
+🤗 Transformers provides a simple and unified way to load pretrained instances, which means you can load an [`AutoModel`] like you would load an [`AutoTokenizer`]. The only difference is selecting the correct [`AutoModel`] for the task. Since we are doing text - or sequence - classification, load [`AutoModelForSequenceClassification`]. The TensorFlow equivalent is simply [`TFAutoModelForSequenceClassification`]:
 
 ```py
->>> for key, value in pt_batch.items():
-...     print(f"{key}: {value.numpy().tolist()}")
-input_ids: [[101, 2057, 2024, 2200, 3407, 2000, 2265, 2017, 1996, 100, 19081, 3075, 1012, 102], [101, 2057, 3246, 2017, 2123, 1005, 1056, 5223, 2009, 1012, 102, 0, 0, 0]]
-attention_mask: [[1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1], [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0]]
+>>> model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
+>>> pt_model = AutoModelForSequenceClassification.from_pretrained(model_name)
 ===PT-TF-SPLIT===
->>> for key, value in tf_batch.items():
-...     print(f"{key}: {value.numpy().tolist()}")
-input_ids: [[101, 2057, 2024, 2200, 3407, 2000, 2265, 2017, 1996, 100, 19081, 3075, 1012, 102], [101, 2057, 3246, 2017, 2123, 1005, 1056, 5223, 2009, 1012, 102, 0, 0, 0]]
-attention_mask: [[1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1], [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0]]
+>>> model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
+>>> # This model only exists in PyTorch, so we use the _from_pt_ flag to import that model in TensorFlow.
```

Comment:
@LysandreJik Can we upload the TF weights of this model on the repo? It would be great to have the doc example work out of the box.

Reply:
Yes, but this will take a bit of time while the authors respond. 

I think it's fine to not slow the PR merge because of this, I'm contacting the authors straight away. (From my side: feel free to merge the PR like this, but I'll let you decide @sgugger)

Reply:
Authors contacted, checkpoint uploaded, feel free to merge